lst_scripts/reorganize_dl1hiperta_to_dl1lstchain.py

```diff
@@ -175,10 +171,6 @@ def modify_params_table(table, position_iterator):
     with np.errstate(invalid='ignore'):
         table.add_column(table['width'] / table['length'], name='wl')
 
-    if position_iterator == 0:
-        print("\n\tRuntime Warnings have been ignored to avoid repeated stdout prints.")
-        print("\tRuntimeWarnings due to invalid values and divide by zero in `log10(intensity)`")
-        print("\t operations and divide by zero in `wl` divisions.")
```

Comment:
I agree that this not super useful, but at least it does reminds the user what's going on... - if ever we are not us the users

Reply:
Well then just don't use `errstate` - the warning should appear only once.

Reply:
Before, due to the `for` for each parameter_table, the message appeared 4 times. 
Indeed, if you can take out the `np.errstate`, the warning should will either in the stdout or in the `*.o` file